### PR TITLE
ARROW-321: fix arrow licenses

### PR DESCRIPTION
**Pull request by julienledem**

### CODE REVIEW

**Review by jaltekruse — APPROVED** [review]
+1

File: dev/release/02-source.sh [hunk 1]
```diff
@@ -56,12 +56,43 @@ tarball=$tag.tar.gz
 # archive (identical hashes) using the scm tag
 git archive $release_hash --prefix $tag/ -o $tarball
 
+# download apache rat
+curl -s https://repo1.maven.org/maven2/org/apache/rat/apache-rat/0.12/apache-rat-0.12.jar > apache-rat-0.12.jar
```
Comment (xhochy) [inline dev/release/02-source.sh]: Could we also run this as part of Travis?

Comment (julienledem) [inline dev/release/02-source.sh]: That would be great.
what's the best way to integrate this into the cpp build?

Comment (xhochy) [inline dev/release/02-source.sh]: Probably the best choice is to just add this command to the travis shell script.

**Pull request closed without merging**